charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -153,7 +153,7 @@ spec:
         - --resources=volumeattachments
 {{  end  }}
 {{ if .Values.namespaces }}
-        - --namespaces={{ .Values.namespaces | join "," }}
+        - --namespaces={{ tpl .Values.namespace $ | join "," }}
```

Comment:
This should probably still be .Values.namespaces since we changed the key name to namespaces